docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/CentOS/README.CentOS7.md

```diff
@@ -13,7 +13,7 @@ Default dependencies installed by CentOS 7.x may not meet the version requiremen
 * MariaDB 10.3 is supported for LORIS 23.* 
 * PHP 7.3 is supported for LORIS 23.*
 
-In addition to the above, the following packages should be installed with `yum` and may also differ from the packages referenced in the main (Ubuntu) [LORIS Readme](./README.md). Detailed command examples are provided below (`sudo` privilege may be required depending on your system).
+In addition to the above, the following packages should be installed with `yum` and may also differ from the packages referenced in the main (Ubuntu) [LORIS Readme](../../../../../README.md). Detailed command examples are provided below (`sudo` privilege may be required depending on your system).
```

Comment:
I think absolute links are better.

Reply:
If someone cloned LORIS, then the link will not point to their fork but back to Aces. That was the point behind relative when it was created.
@driusan @ridz1208 What do you think?

Reply:
Absolute links are always wrong.